### PR TITLE
🎨 Link keynote to the speakers page

### PR DESCRIPTION
**Pull request by tngTUDOR**

### CODE REVIEW

**Comment by tngTUDOR** [comment]
@LaurentChion , I re-used the "Quotes" component, but we might want to create a new one, where we can add a "resume" of the keynote presentation inside the card, as well as changing the background color, since this are note testimonials, but keynote speaker + summary

File: components/Conference.vue [hunk 1]
```diff
@@ -20,145 +20,167 @@
             </tr>
           </thead>
           <tbody>
-            <tr :key="j" v-for="(session, j) in day.sessions">
-              <td>{{ session.time }}</td>
-              <td>{{ session.title }}</td>
-              <td>{{ session.presenters }}</td>
-            </tr>
+            <template v-for="session in day.sessions">
```
Comment (ok-lch) [inline components/Conference.vue]: Pourquoi un template ?

**Review by ok-lch — COMMENTED** [review]
One thing to change on Conference.vue

File: components/Conference.vue (hunk 1 above)
Comment (tngTUDOR) [reply components/Conference.vue]: Pour pouvoir faire le conditional formatting selon si c'est un keynote ou pas

**Pull request merged**